app/models/fog_area.rb

```diff
@@ -0,0 +1,8 @@
+class FogArea < ApplicationRecord
```

Comment:
The naming of this class is interesting. To me, "Fog Area" implies "The area where there is fog", but in reality, fog areas are the place where there is no fig, right? 

The Fog covers the whole map, and these are the areas that are revealed. Would something like `RevealArea` or `RevealedArea`, or even `VisibleArea` be a better name?

Reply:
I think this naming would then cascade throughout the code where areas are being drawn called "fog areas", but then they have the class`fog__revealed` on them...

Reply:
> The naming of this class is interesting. To me, "Fog Area" implies "The area where there is fog", but in reality, fog areas are the place where there is no fig, right?

Yes, that's an excellent point! I hadn't considered it in that way.

Whilst I was playing with these, I thought it might make sense to be enable to toggle the areas as hidden/visible:

- A DM revealing areas live can just draw sections to reveal
- A DM preparing maps in advance and draw out sections then reveal them as they play

If we go in that direction, a general name like `FogArea` name with a `revealed` boolean might make sense?

The other consideration is that we might need to draw areas to hide as well, if too much is revealed by mistake or the characters should no longer be able to view a room. If that were the case then this would imply we'd need a corresponding `HiddenArea` or get into some complex path intersection stuff.

So either way I think it would be useful to have a name which conveys that:

- this model indicates a geometry object related to the fog layer
- this model could either show or hide an area of fog